lib/request.js

```diff
@@ -257,15 +257,14 @@ module.exports = {
    */
 
   get fresh() {
-    const method = this.method;
-    const s = this.ctx.status;
+    const { method, ctx: { status: s, response }, header } = this;
```

Comment:
Destruction is not a very readable thing if you are doing Javascript for last 15 years. Nor Lisp is readable for me. But for somebody who is starting to develop or contribute to Koa in 2017 and agreed with node 7.6+ requirement, destruction is perfectly readable and acceptable... It's not about comfort level, that's native language features now.